### ISSUE TITLE
Question: how to use  RecurrenceRuleScribe scribe = new RecurrenceRuleScribe();

### ISSUE THREAD

**Issue by Kashif2016** [mangstadt/biweekly#65]
@mangstadt  thankyou for this library.
I am new to android so sorry if my question is really basic. 
i am trying to use your library to parse RRULE of my ics file, so i looked into FAQ and Wiki, how to use RecurrenceRuleScribe scribe = new RecurrenceRuleScribe();  but when i add this in my code in "mainClass" i get an error

 ```
Caused by: java.lang.NoClassDefFoundError: com.github.mangstadt.vinnie.SyntaxStyle
 at biweekly.ICalVersion.<clinit>(ICalVersion.java:39)
 at biweekly.io.scribe.property.ICalPropertyScribe.<clinit>(ICalPropertyScribe.java:75)

```
i have already added biweekly-0.6.0.jar in libs and imports in "main class".
```
import biweekly.io.scribe.property.RecurrenceRuleScribe;
import biweekly.parameter.ICalParameters;
import biweekly.property.DateEnd;
import biweekly.property.DateStart;
import biweekly.property.DateTimeStamp;
import biweekly.property.RecurrenceRule;

```
i have already get the Start Date , End Date and RRULE:FREQ=MONTHLY;BYDAY=FR;BYMONTHDAY=13
just need to parse this RRULE to get all the events .... 
It would be really appreciated If you could please guide me how i can parse this RRULE

**Comment by mangstadt** [comment]
You must also have the "vinnie" library on your classpath.

You can get the vinnie library by going to the [Downloads](https://github.com/mangstadt/biweekly/wiki/Downloads) page and clicking on the "JAR with Dependencies" link.

**Comment by Kashif2016** [comment]
Thank you for your quick reply , i'll check that..

**Comment by Kashif2016** [comment]
Thank you very much it is working now.

how can i pass it my start date , end date and RRULE to get all the events?

**Comment by mangstadt** [comment]
```java
Recurrence recur = ...
Date start = ...
TimeZone yourTimeZone = ...
RecurrenceRule rrule = new RecurrenceRule(recur);
DateIterator it = rrule.getDateIterator(start, yourTimeZone);
```

**Comment by Kashif2016** [comment]
thank you ill try it

**Comment by Kashif2016** [comment]
I have tried a lot but app get crashed on first line 
```
Recurrence recur = new Recurrence.Builder(Frequency.valueOf(component.getProperty("RRULE").getValue())).build();
Date start = checkStartDate;
TimeZone TZ = TimeZone.getDefault();
RecurrenceRule rrule = new RecurrenceRule(recur);
DateIterator it = rrule.getDateIterator(start,TZ);

```

P.S 
component.getProperty("RRULE").getValue() =>RRULE:FREQ=MONTHLY;BYDAY=FR;BYMONTHDAY=13

**Comment by Kashif2016** [comment]
i have found a solution in one of your closed issue #47 
now i am able to display all the recurring events.
Thanks for the library
